spec/index.html

```diff
@@ -270,12 +270,44 @@
 <body>
   <section id='abstract'>
     <p>
-      In recent years, [[YAML]] has emerged as a more concise format
-      to represent information that had previously been serialized as JSON,
-      including Linked Data.
-      This document defines how to serialize linked data
-      in YAML.
-      Moreover, it registers the `application/ld+yaml` media type.
+      <strong>Objective.</strong>
+      This document defines YAML-LD, a set of conventions built on top
+      of YAML, which outlines how to serialize Linked Data as YAML
+      based on JSON-LD syntax, semantics, and APIs. The emergence of
+      YAML as a more concise format for representing information
+      otherwise serialized as JSON, including Linked Data, has led to
+      the development of YAML-LD.
+    </p>
+
+    <p>
+      <strong>Methods.</strong>
+      The document defines constraints on YAML so that any YAML-LD
+      document can be represented in JSON-LD. This is because YAML is
+      more expressive than JSON, both in terms of available data types
+      and the document structure. The document also registers the
+      <code>application/ld+yaml</code> media type.
```

Comment:
```suggestion
      This document defines constraints on YAML so that any YAML-LD
      document can be represented in JSON-LD. This is necessary
      because YAML is more expressive than JSON, in terms of both
      available data types and document structure. This document
      also registers the <code>application/ld+yaml</code> media type.
```